lib/internal/test_runner/mock.js

```diff
@@ -131,8 +132,13 @@ class MockTracker {
     implementation = kDefaultFunction,
     options = kEmptyObject,
   ) {
-    validateObject(object, 'object');
     validateStringOrSymbol(methodName, 'methodName');
+    // In case object is a static class
+    if (typeof object === 'function') {
+      validateFunction(object, methodName);
```

Comment:
No need to have this line, since `validateFunction` only checks for the typeof object passed.

Reply:
so in this case, if `object` is a function I won't use the `validateObject` function? (otherwise the code will crash)

Reply:
I think the following is sufficient:

```
if (typeof object !== 'function')
  validateObject(object, 'object')
```

Reply:
got it!